yarn.lock

```diff
@@ -1828,6 +1833,19 @@
     semver "^7.3.7"
     tsutils "^3.21.0"
 
+"@typescript-eslint/typescript-estree@5.43.1-alpha.22":
```

Comment:
Is it expected to see this version alongside `"@typescript-eslint/typescript-estree@5.20.0"` (which is also present in `yarn.lock`)? Just checking if we need to `yarn dedupe` or keep things as is.

Reply:
`yarn dedupe` not available in `main` branch.

Reply:
But we should dedupe before merge.

Reply:
We can use [yarn-deduplicate](https://www.npmjs.com/package/yarn-deduplicate) or just manually edit `yarn.lock`. Removing duplicate older entries manually and running `yarn install` may work.

Reply:
FYI Some other packages are duduped by `npx yarn-dedupulicate`. [90d6098](https://github.com/prettier/prettier/pull/13872/commits/90d6098cf250ec25714e10b15ba39594c24d04fe)

Reply:
Nice! The majority of deduplicated deps are related to tests and linting, which is good. I hope to see automatic deduplication in Yarn Berry one day: https://github.com/yarnpkg/berry/issues/4976 🤞

Reply:
https://github.com/typescript-eslint/typescript-eslint/releases/tag/v5.44.0 is out 🕺